src/script/lua_api/l_object.cpp

```diff
@@ -1410,13 +1402,25 @@ int ObjectRef::l_get_player_control(lua_State *L)
 	lua_setfield(L, -2, "dig");
 	lua_pushboolean(L, control.place);
 	lua_setfield(L, -2, "place");
+	lua_pushnumber(L, control.movement_speed);
+	lua_setfield(L, -2, "movement_speed");
+	lua_pushnumber(L, control.movement_direction);
+	lua_setfield(L, -2, "movement_direction");
```

Comment:
No, not correct. Like the comment which I suggested adding explains these variables are not available on the server.

Reply:
I removed them again, and just kept the `control.up` to `player->keyPressed & (1 << 0)` changes needed for compillation